buildSrc/src/main/java/baritone/gradle/task/BaritoneGradleTask.java

```diff
@@ -49,8 +49,8 @@ class BaritoneGradleTask extends DefaultTask {
             ARTIFACT_STANDALONE         = "%s-standalone-%s.jar",
             ARTIFACT_FORGE_UNOPTIMIZED  = "%s-forge-unoptimized-%s.jar",
             ARTIFACT_FORGE_API          = "%s-forge-api-%s.jar",
-            ARTIFACT_FORGE_STANDALONE   = "%s-forge-standalone-%s.jar",
-            ARTIFACT_FABRIC_UNOPTIMIZED = "%s-fabric-standalone-%s.jar",
```

Comment:
Why have you removed forge and fabric from the names here?

Reply:
o but I thought I undid that as I handled it differently below, good catch